app/v2/partials/directives/contribution-form.html

```diff
@@ -65,7 +65,30 @@ <h2 class="modal__title" ng-if="isIdea">{{actionLabel | translate}} {{'Idea' | t
             </div>
           </fieldset>
         </div>
+        <div ng-init="addFile=false">
+          <fieldset>
+            <a href="" ng-click="addFile = !addFile"><i class="fa fa-paperclip"></i>&nbsp;{{'Add files' | translate}}</a>
+          </fieldset>
+        </div>
+        <div ng-show="addFile">
+          <fieldset>
+            <div class="field__container">
+              <label for="name">{{'Name' | translate}}</label>
+              <input name="name" ng-model="newAttachment.name">
+            </div>
+          </fieldset>
+          <fieldset>
+            <div class="field__container">
+              <label for="file">{{'File' | translate}}</label>
+              <input type="file" name="file" fileread="newAttachment.file">
+            </div>
+          </fieldset>
+          <fieldset>
+            <button class="button__modal_submit" ng-click="submitAttachment()">{{'Add' | translate}}</button>
```

Comment:
Let's change to "Add file" and then hide the form to add the file, replacing it by the Name of the file with small button to delete it (which should just remove that resource from the attachments array.